Library/Homebrew/test/cmd/fetch_spec.rb

```diff
@@ -14,4 +14,24 @@
     expect(HOMEBREW_CACHE/"testball--0.1.tbz").to be_a_symlink
     expect(HOMEBREW_CACHE/"testball--0.1.tbz").to exist
   end
+
+  it "concurrently downloads formula URLs", :integration_test do
+    setup_test_formula "testball1"
+    setup_test_formula "testball2"
+
+    expect { brew "fetch", "testball1", "testball2", "--concurrency=2" }.to be_a_success
+
+    expect(HOMEBREW_CACHE/"testball1--0.1.tbz").to be_a_symlink
+    expect(HOMEBREW_CACHE/"testball1--0.1.tbz").to exist
+    expect(HOMEBREW_CACHE/"testball2--0.1.tbz").to be_a_symlink
+    expect(HOMEBREW_CACHE/"testball2--0.1.tbz").to exist
+  end
+
+  it "errors when concurrently downloading to the same destination", :integration_test do
```

Comment:
Can probably skip this given it's not important functionality and we don't really run into this case in Homebrew/homebrew-core? Don't feel strongly, though.

Reply:
What's the actual error here?

Reply:
I realized after playing with concurrency that this test has add'l value - it will fail if the fetch is performed non-concurrently (er, serially). I'm inclined to keep the test for that reason, wdyt?

Reply:
Cool, works for me.